packages/nimble-components/src/rich-text-mention/base/models/mention-validator.ts

```diff
@@ -144,7 +145,8 @@ export class RichTextMentionValidator<
 
     private isInvalidRegex(pattern: string): boolean {
         try {
-            RegExp(pattern);
+            // eslint-disable-next-line no-new
```

Comment:
ESLint disables should have an explanation justifying them (or even better, be fixed not to violate the rule). In this case if the goal is to see whether the constructor throws then you could add a comment saying that.

Reply:
Added comment line